.github/workflows/ci.yml

```diff
@@ -64,21 +64,20 @@ jobs:
         env:
           CACHE_KEY: "matrix-setup"
         run: echo "$CACHE_KEY" > .gocache.tmp
-      - uses: actions/setup-go@v3 # only used by gotestsum
+      - name: Setup Go Caching
+        uses: actions/setup-go@v3 # only used by gotestsum
         with:
           go-version: '>=1.19.0' # decoupled from version sets, only used by gotestsum
           cache: true
           cache-dependency-path: |
             pkg/go.sum
             .gocache.tmp
-      - uses: actions/checkout@v3
+      - name: Install gotestsum
+        uses: jaxxstorm/action-install-gh-release@v1.7.1
         with:
-          repository: dnephin/gotestsum
-          ref: d09768c81065b404caed0855eb3ab8f11a2a4431
-          path: vendor/gotestsum
-      - run: |
-          cd vendor/gotestsum
-          go install .
+          repo: gotestyourself/gotestsum
+          tag: v1.8.1
+          cache: enable
```

Comment:
I have a local change that attempted to do this too.
We can't switch to this until https://github.com/gotestyourself/gotestsum/pull/277 is released
since we're using that flag in get-job-matrix.py

https://github.com/pulumi/pulumi/blob/mckinstry/speedup-lint/scripts/get-job-matrix.py#L285